src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -1,6 +1,8 @@
 // Copyright 2022 - 2024, University of Freiburg
 // Chair of Algorithms and Data Structures
-// Author: Johannes Kalmbach <kalmbach@cs.uni-freiburg.de>
+// Authors: Johannes Kalmbach <kalmbach@cs.uni-freiburg.de>
+//          Robin Textor-Falconi <robintf@cs.uni-freiburg.de>
```

Comment:
```suggestion
//          Robin Textor-Falconi <textorr@informatik.uni-freiburg.de>
```

I don't think this other email does exist tbh.